stripe/_encode.py

```diff
@@ -0,0 +1,45 @@
+import calendar
+import datetime
+import time
+from collections import OrderedDict
+from typing import Generator, Tuple, Any
+
+
+def _api_encode(data) -> Generator[Tuple[str, Any], None, None]:
```

Comment:
Any reason for moving these functions out of _api_requestor.py (I'm a fan for cleanliness though)? Would this be breaking if a user were trying to do `from stripe.api_requestor import _encode_datetime` or something?

Reply:
These helpers don't have any dependencies and loading the entire `stripe.api_requestor ` module just to get the helper caused circular references.

All names that start with underscore are considered private so it feels like a fair game to move these helpers.